### PR TITLE
add a version command and initial release

### PR DESCRIPTION
**Pull request by BenTheElder**
this will let us create the first "release" of kind, with a version

### CODE REVIEW

**Comment by BenTheElder** [comment]
fixes #140

File: cmd/kind/version/version.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+// Package version implements the `version` command
+package version
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// Version is the kind CLI version
+const Version = "0.1.0-alpha0.0.10.0.1-alpha"
```
Comment (munnerz) [inline cmd/kind/version/version.go]: 🤔 this version number looks wrong?

Comment (BenTheElder) [reply cmd/kind/version/version.go]: bad sed :D

Comment (BenTheElder) [reply cmd/kind/version/version.go]: fixed

**Comment by munnerz** [comment]
I'll leave you to `/hold cancel` if you want to get this in 😄 

/lgtm
/hold

**Comment by BenTheElder** [comment]
let's do it!
/hold cancel

File: hack/release/create.sh [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+# creates a release and following pre-release commit for `kind`
+# builds binaries between the commits
+# Use like: create.sh <release-version> <next-prerelease-version>
+# EG: create.sh 0.0.1 0.1.0-alpha
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [ "$#" -ne 2 ]; then
+    echo "Usage: create.sh relase-version next-prerelease-version"
```
[the remark this answers is not in the document]
Comment (BenTheElder) [reply hack/release/create.sh]: filed a fix https://github.com/kubernetes-sigs/kind/pull/142#issue-234472979

**Pull request merged**